Model/Behavior/AuditableBehavior.php

```diff
@@ -173,6 +174,11 @@ public function afterSave(Model $Model, $created, $options = array()) {
 			array('hasMany' => array('AuditLog.Audit'))
 		);
 
+		if(isset($this->settings[$Model->alias]['data_source'])) {
```

Comment:
This could be extracted into a separate protected method. Would increase readability, reduce method length and complexity and would enable one to overwrite that new method, if needed.